Source/Core/Common/CommonPaths.h

```diff
@@ -7,26 +7,44 @@
 #define DIR_SEP "/"
 #define DIR_SEP_CHR '/'
 
-// The user data dir
+// The current working directory
 #define ROOT_DIR "."
+
+// The normal user directory
+#ifndef STEAM
 #ifdef _WIN32
-#define USERDATA_DIR "User"
-#define DOLPHIN_DATA_DIR "Dolphin"
-#elif defined __APPLE__
-// On OS X, USERDATA_DIR exists within the .app, but *always* reference
-// the copy in Application Support instead! (Copied on first run)
-// You can use the File::GetUserPath() util for this
-#define USERDATA_DIR "Contents/Resources/User"
-#define DOLPHIN_DATA_DIR "Library/Application Support/Dolphin"
-#elif defined ANDROID
-#define USERDATA_DIR "user"
-#define DOLPHIN_DATA_DIR "/sdcard/dolphin-emu"
-#define NOMEDIA_FILE ".nomedia"
+#define NORMAL_USER_DIR "Dolphin Emulator"
+#elif defined(__APPLE__)
+#define NORMAL_USER_DIR "Library/Application Support/Dolphin"
```

Comment:
```suggestion
#define NORMAL_USER_DIR "Library/Application Support/Dolphin Emulator"
```

Reply:
We can't do this, otherwise we will break existing installations on macOS. Dolphin will try to look for a directory named `Dolphin Emulator`, find it doesn't exist, and create a whole new folder. To the user, it would look like Dolphin deleted all their save data and configs.